### PR TITLE
[Drawer] - Remove setClass API

### PR DESCRIPTION
**Pull request by bluong**

### CODE REVIEW

File: src/drawers/arcDrawer.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export module Drawers {
 
     constructor(dataset: Dataset) {
       super(dataset);
+      this._className = "arc";
```
Comment (aicioara) [inline src/drawers/arcDrawer.ts]: Is this ever needed?

**Comment by aicioara** [comment]
Rest looks fine

**Comment by jtlan** [comment]
Hold on, does this remove the "arc" class?

File: src/drawers/arcDrawer.ts (hunk 1 above)
Comment (bluong) [inline src/drawers/arcDrawer.ts]: The class name is added here to specify what types of paths an Arc Drawer will draw.  Note that Symbol Drawer labels with a "symbol" class

Comment (aicioara) [inline src/drawers/arcDrawer.ts]: Is there a reason why `Drawers.Rect` does not have a class associated with it?

Comment (bluong) [inline src/drawers/arcDrawer.ts]: The types of "rect"s that are drawn in a `Drawers.Rect` are... "rect"s?  It is really redundant I think in this case, whereas we have many different types of "path"s with no good way of differentiation

**Pull request merged**